projects/ngx-quill/src/lib/quill-editor.component.ts

```diff
@@ -129,14 +111,14 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
   onValidatorChanged: () => void
 
   constructor(
-    @Inject(ElementRef) private elementRef: ElementRef,
```

Comment:
`QuillEditorBase` constructor parameter accessibility modifiers have simply been changed from `private` to `protected` to allow for their access by derived classes / components.

`Inject` decorators have been moved back to `QuillEditorComponent` below.